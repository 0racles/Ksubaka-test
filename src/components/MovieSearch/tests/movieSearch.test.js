import { MovieSearch } from '../index';

const props = {
  onKeyPress: (f) => f,
};

describe('Movie Search', () => {
  it('should match snapshot', () => {
    const renderedItem = shallow(<MovieSearch {...props} />);
    expect(renderedItem).toMatchSnapshot();
  });
});
