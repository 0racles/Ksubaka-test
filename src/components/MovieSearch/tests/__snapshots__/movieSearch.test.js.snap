// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Search should match snapshot 1`] = `
<.sc-bdVaJa
  className="movieSearch"
>
  <.sc-bwzfXH
    className="movieSearch-input"
    onKeyPress={[Function]}
    placeholder="enter movie title..."
    type="text"
  />
  <.sc-htpNat
    className="movieSearch-button"
    onClick={[Function]}
    type="button"
    value="Search"
  />
</.sc-bdVaJa>
`;
